content/pt/docs/contributing/sig-practices.md

```diff
@@ -4,12 +4,12 @@ linkTitle: Práticas do SIG
 description:
   Saiba como aprovadores e mantenedores gerenciam issues e contribuições.
 weight: 999
-cSpell:ignore: chalin Comms docsy onboarded
+cSpell:ignore: chalin Comms docsy onboarded mergeados branch
 ---
 
 Esta página inclui diretrizes e algumas práticas comuns utilizadas por aprovadores e mantenedores.
 
-## Integração (Onboarding)
+## Integração (Onboarding) {#onboarding}  <!-- ADDED ANCHOR HERE -->
```

Comment:
please, do not add comments explaining the changes you did. This is valid for this and for all other comments you added below